pkg/plugins/golang/declarative/v1/api.go

```diff
@@ -134,10 +134,13 @@ func (p *createAPISubcommand) Scaffold(fs machinery.Filesystem) error {
 	}
 
 	// Ensure that we are pinning sigs.k8s.io/kubebuilder-declarative-pattern version
-	kbDeclarativePattern := kbDeclarativePatternForV2
+	// Just pin an old value for go/v2. It shows fine for now. However, we should improve/change it
+	// if we see that more rules based on the plugins version are required.
+	kbDeclarativePattern := kbDeclarativePatternForV3
 	for _, pluginKey := range p.config.GetPluginChain() {
-		if pluginKey == plugin.KeyFor(goPluginV3.Plugin{}) {
-			kbDeclarativePattern = kbDeclarativePatternForV3
+		if pluginKey == plugin.KeyFor(goPluginV2.Plugin{}) || pluginKey == "go.kubebuilder.io/v2" {
```

Comment:
There is no difference between the output of `KeyFor(goPluginV2.Plugin{})` and this string

Reply:
@camilamacedo86 the go/v2 plugin no longer has “base” in the name right?